source/postprocess/visualization/volume_of_fluid_values.cc

```diff
@@ -32,7 +32,8 @@ namespace aspect
       VolumeOfFluidValues<dim>::
       VolumeOfFluidValues ()
         :
-        DataPostprocessor<dim> ()
+        DataPostprocessor<dim> (),
+        Interface<dim>("m*m*m")
```

Comment:
I believe that this postprocessor outputs unitless numbers (despite the occurrence of the word "Volume" in the name of the class):
```suggestion
        Interface<dim>("")
```